packages/coinstac-client-core/test/coinstac-client.js

```diff
@@ -213,3 +213,33 @@ test('CoinstacClient - teardown process', (t) => {
     .catch(t.end);
 });
 
+test('CoinstacClient - auth initialization', t => {
```

Comment:
Ensures login is called when creating a user. Not the greatest test, I think, but it works.